.SAMPLE_env

```diff
@@ -1,2 +1,8 @@
+// Set which model provider you want to use, HUGGING_FACE or OPEN_AI
+ENABLED_MODEL_STORE="HUGGING_FACE"
```

Comment:
Should this have quotes around it? Not sure but based on my other env files it feels like no.